### ISSUE TITLE
Show module hint when auto-importing

### ISSUE THREAD

**Issue by santiagocezar** [microsoft/pylance-release#2885]
Probably a duplicate of #2604. That issue was closed with the conclusion that it was a problem in VSCode UX, but other extensions have this feature already so that's not it.

Usually, I need to use `typing.cast`, but VSC shows 2 functions called cast, and I have no way to know which one is it.
![Captura desde 2022-06-01 20-00-34](https://user-images.githubusercontent.com/48573316/171649757-6c7fe8ea-9c41-4284-aaf7-d0a967511196.png)

also the first one always ends up being wrong, even though I've already imported something from `typing`
![Captura desde 2022-06-01 20-00-50](https://user-images.githubusercontent.com/48573316/171650983-9fef5648-7c10-4b01-8b25-2d668e4fc365.png)

I'm asking for something like what the Typescript extension does, it shows a hint besides the identifier with the name of the library.
![Captura desde 2022-06-01 20-06-12](https://user-images.githubusercontent.com/48573316/171652468-e23e0488-090f-4b9f-8949-9d13a34f2a79.png)

rust-analyzer also does this, and probably many others but those 2 are the ones that I tested.

Hopefully it's easy to implement! Good luck!

**Comment by erictraut** [comment]
A module hint is provided, but you need to ask VS Code to display expanded information for the suggestion. This is true for other language servers as well, not something that's unique to Pylance.

To show the expanded information, move your mouse pointer over the suggestion, and you'll see a `>` icon. Click it, and you'll see a separate window to the right of the primary completion list window. Once you toggle this, it will appear every time you invoke completion suggestions, so you need to do this only once.

![Screen Shot 2022-06-02 at 7 41 31 AM](https://user-images.githubusercontent.com/7040122/171655891-66010c65-23f5-46bc-bf3d-475fdb461bf7.png)

![Screen Shot 2022-06-02 at 7 41 37 AM](https://user-images.githubusercontent.com/7040122/171655910-e962bed3-dfca-4331-96d9-b906afb6ac62.png)

**Comment by santiagocezar** [comment]
Yes I know, but I learned about it in that other issue, so I'm not sure if it's really discoverable.

Something like this mockup should be better IMO
![mockup](https://user-images.githubusercontent.com/48573316/171677506-5ee87f82-ca33-4a5f-ad40-e5dc107c5751.png)

and instead of showing the import statement in that popup, show information about the function
![Captura desde 2022-06-02 13-20-16](https://user-images.githubusercontent.com/48573316/171678968-a63ce6f4-c553-48e2-9aeb-9070589f6a13.png)

That's how the Typescript extension works basically, which I think it's a very good example and I hope you agree

**Comment by erictraut** [comment]
Actually, Pylance is consistent with how TypeScript and other language servers work in this regard. Here's a screen shot that shows an auto-import in TypeScript.

![Screen Shot 2022-06-02 at 9 39 12 AM](https://user-images.githubusercontent.com/7040122/171680731-98aed919-e850-4a2d-bef9-5ead0c2b9523.png)

I think you make a good point about discoverability.

**Comment by santiagocezar** [comment]
Weird?... This is how it looks for me (VSCode 1.67.1)
![imagen](https://user-images.githubusercontent.com/48573316/171682093-cfa76dac-8f41-4e4c-8f2a-51511fda473d.png)

And btw, thank you for the really quick feedback!

**Comment by erictraut** [comment]
Yeah, you're right. I see that for some TypeScript auto-imports but not for all. We'll need to dig into it further to understand what's going on here.

Thanks for the feedback.

**Comment by santiagocezar** [comment]
You're welcome!

**Comment by debonte** [comment]
This issue has been fixed in version 2022.6.10, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2022610-8-June-2022)
